dspace-server-webapp/src/test/java/org/dspace/app/rest/RelationshipDeleteRestRepositoryIT.java

```diff
@@ -0,0 +1,574 @@
+
+        publicationItem = itemService.find(context, publicationItem.getID());
+        List<MetadataValue> publicationAuthorList =
+            itemService.getMetadata(publicationItem, "dc", "contributor", "author", Item.ANY);
+        assertThat(publicationAuthorList.size(), equalTo(1));
+        assertThat(publicationAuthorList.get(0).getValue(), equalTo("Smith, Donald"));
+        assertThat(publicationAuthorList.get(0).getAuthority(), startsWith("virtual::"));
+        List<MetadataValue> publicationRelationships = itemService.getMetadata(publicationItem,
+            "relation", "isAuthorOfPublication", Item.ANY, Item.ANY);
+        assertThat(publicationRelationships.size(), equalTo(1));
+
+        projectItem = itemService.find(context, projectItem.getID());
+        List<MetadataValue> projectAuthorList =
+            itemService.getMetadata(projectItem, "dc", "contributor", "author", Item.ANY);
+        assertThat(projectAuthorList.size(), equalTo(1));
+        assertThat(projectAuthorList.get(0).getValue(), equalTo("Smith, Donald"));
+        assertThat(projectAuthorList.get(0).getAuthority(), startsWith("virtual::"));
+        List<MetadataValue> projectRelationships = itemService.getMetadata(projectItem,
+            "relation", "isPersonOfProject", Item.ANY, Item.ANY);
+        assertThat(projectRelationships.size(), equalTo(1));
+    }
+
+    @Test
+    public void testDeleteAuthorRelationshipCopyToLeftItem() throws Exception {
+        initPublicationAuthor();
+
+        getClient(adminAuthToken).perform(
+            delete("/api/core/relationships/" + relationship.getID() + "?copyVirtualMetadata=left"))
+            .andExpect(status().isNoContent());
+
+        leftItem = itemService.find(context, leftItem.getID());
+        List<MetadataValue> authorList = itemService.getMetadata(leftItem, "dc", "contributor", "author", Item.ANY);
+        assertThat(authorList.size(), equalTo(1));
+        assertThat(authorList.get(0).getValue(), equalTo("familyName, firstName"));
+        assertThat(authorList.get(0).getMetadataField().getMetadataSchema().getName(), equalTo("dc"));
+        assertThat(authorList.get(0).getMetadataField().getElement(), equalTo("contributor"));
+        assertThat(authorList.get(0).getMetadataField().getQualifier(), equalTo("author"));
+        assertNull(authorList.get(0).getAuthority());
+
+        List<MetadataValue> relationshipMetadataList = itemService
+            .getMetadata(leftItem, "relation", "isAuthorOfPublication", null, Item.ANY);
+        assertThat(relationshipMetadataList.size(), equalTo(0));
+    }
+
+    @Test
+    public void testAuthorDeleteRelationshipCopyToRightItem() throws Exception {
+        initPublicationAuthor();
+
+        getClient(adminAuthToken).perform(
+            delete("/api/core/relationships/" + relationship.getID() + "?copyVirtualMetadata=right"))
+            .andExpect(status().isNoContent());
+
+        leftItem = itemService.find(context, leftItem.getID());
```

Comment:
This looks to be testing the wrong Item?  It says it should test Right item, but it looks to be testing `leftItem`

Reply:
This is a setup where the Right item doesn't contain virtual metadata.
You can see a setup with bi-directional virtual metadata in the journal tests further down in this class
A comment explaining this has been included